dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_h2.sql

```diff
@@ -15,13 +15,13 @@
  * limitations under the License.
 */
 
-SET
-FOREIGN_KEY_CHECKS=0;
+SET FOREIGN_KEY_CHECKS=0;
+SET REFERENTIAL_INTEGRITY FALSE;
 
 -- ----------------------------
 -- Table structure for QRTZ_JOB_DETAILS
 -- ----------------------------
-DROP TABLE IF EXISTS QRTZ_JOB_DETAILS;
+DROP TABLE IF EXISTS QRTZ_JOB_DETAILS CASCADE;
```

Comment:
Do the other sql files should be modified too?

Reply:
No. The other files has something like `set foreignkeyconstraintcheck false`

Reply:
@zhongjiajie BTW, the mergeable bot doesn't detect this, I only modified `_h2.sql` in this PR but it still passes, although the others don't need to modified in sync in this PR

Reply:
@kezhenxu94 Thanks for pointing it out, I would take a look on this